### PR TITLE
Updated link for PyMacAdmin

### PR DESCRIPTION
**Pull request by benroeder**
https://code.google.com/p/pymacadmin/ ->https://github.com/nigelkersten/pymacadmin

### CODE REVIEW

**Comment by benroeder** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ This is not an official Google product.
   [Munki]: http://code.google.com/p/munki
   [Santa]: https://github.com/google/santa
   [Cauliflower Vest]: https://github.com/google/cauliflowervest
-  [PyMacAdmin]: http://code.google.com/p/pymacadmin
+  [PyMacAdmin]: https://github.com/nigelkersten/pymacadmin
```
Comment (ccaviness) [inline README.md]: https://github.com/MacSysadmin/pymacadmin

**Comment by ccaviness** [comment]
Thanks for the pull request - can you update it to https://github.com/MacSysadmin/pymacadmin instead?

**Comment by benroeder** [comment]
Updated as requested and updated the munki link as well

**Pull request merged**